src/v/kafka/server/handlers/create_topics.cc

```diff
@@ -244,18 +244,7 @@ ss::future<response_ptr> create_topics_handler::handle(
     valid_range_end = quota_exceeded_it;
 
     auto to_create = to_cluster_type(begin, valid_range_end);
-    /**
-     * We always override cleanup policy. i.e. topic cleanup policy will
-     * stay the same even if it was changed in defaults (broker
-     * configuration) and there was no override passed by client while
-     * creating a topic. The the same policy is applied in Kafka.
```

Comment:
Was this comment just wrong, or did Kafka change?

Reply:
Not sure, however I confirmed in Kafka 3.3 that this is no longer the case